### ISSUE TITLE
<!DOCTYPE html> <html lang="en"> <head> <meta charset="UTF-8"> <title> WILL YOU BE MY VALENTINE?</title> <meta name="view port" content= "width=device-width, user-scalable=no, initial scale =1.0, maximum-scale=1.0, minimum scale=1.0" > <link rel="stylesheet" href="main.css> <script src= "https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js" ></script> <script> $(document).ready(function(){ $(' .p1').hover(function(){ arriba = math.random()*(400-1) +1; abajo = math.random() *  (609-1) *1; $(this).css('top' , arriba); $(this).css('left ' , abajo); }); }); function  dijoSi(){ document.getElementById('si').style.display  = 'block'; } </script> </head> <body> <div class="contenedor"> <div class="titulo"> <h1> WILL YOU BE MY VALENTINE?</h1> </div> <div class="opciones"> <p class="p1">no</p> <p onclick="dijoSi()" class "p2">si</p> <div id="si"> <h2> TE AMO MUCHO MI AMOISHITA</h2> </div> </div> </div> </body> </html>

### ISSUE THREAD

**Issue by dgrmdss** [avelino/awesome-go#5589]
*}
margin: 0px; 
padding: 0px;
}
body{
background: #fcfcfc;
}
.contenedor{
 margin: auto; 
width: 900px;
margin-top: 300px; 
text-align: center;
}
.titulo margin-bottom: 20px;
}
•opciones p{
background: black; 
display: initial; 
padding: 12px; 
color: #FF00FF; 
border-radius: 2px; 
font-weight: bold;
}
p.p1{
position: absolute;
top: 400px; 
left: 500px; 
width: 100px;
}
p.p1: hover{
background: pink; 
cursor: pointer;
}
p.p2{
position: absolute;
top: 400px; 
left: 750px; 
width: 100px;
}
p.p2:hover{
background: pink; 
cursor: pointer;
}
#si{
background: #pink;
left: 420px;
position: absolute;
top: 122px; 
width: 550px;
color: #FF00FF; 
padding: 50px; 
display: none;
}~
